### DISCUSSION TITLE
Disable Checking Temporarily

### DISCUSSION THREAD

**Discussion by ZikangXiong** [znck/grammarly#251] [Ideas]
Thank you very much for this awesome extension! 

After bumping from version 14.1 to 18.1, it seems that we cannot disable the checking temporarily with a simple click. Usually, I only enable Grammarly after writing down a whole file to avoid distraction and reduce resource usage, so temporarily disabling the checking and enabling only when necessary will be very helpful. Could you bring this function back?

**Comment by znck** [comment]
Done #254
